src/site/content/en/blog/compat2021-holiday-update/index.md

```diff
@@ -0,0 +1,150 @@
+ have landed in WebKit, leading to a 1% improvement for Firefox and 3% improvement for Safari on 
+the Grid tests.
+
+
+## CSS `position: sticky`
+
+In our last update, we noted that `position: sticky` was the first area where any browser (in this 
+case Chrome and Edge) reached 100% passing tests. Now, following a [number of fixes](https://bugs.webkit.org/buglist.cgi?bug_status=RESOLVED&email1=mrobinson%40webkit.org&emailassigned_to1=1&emailtype1=equals&list_id=7731158&query_format=advanced&resolution=FIXED&short_desc=sticky&short_desc_type=allwordssubstr) 
+in WebKit’s implementation, Safari also scores 100% for these tests.  Most of these improvements 
+were included in Safari 15.
+
+
+## CSS `aspect-ratio` property
+
+Cross-browser support for defining the aspect ratio (width-to-height ratio) of elements has 
+continued to improve, with Compat 2021 scores reaching 99%, 97% and 95% for Chrome/Edge, Firefox 
+and Safari respectively. Most of the improvements are not with the `aspect-ratio` property itself, 
+but rather with how `width` and `height` attributes are [mapped to a default `aspect-ratio` value](https://developer.mozilla.org/docs/Web/Media/images/aspect_ratio_mapping) 
+for elements. This was implemented for multiple elements in [WebKit](https://wpt.fyi/results/html/rendering/replaced-elements/attributes-for-embedded-content-and-images?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5739124314079232&run_id=6290692121821184), 
+and `<canvas>` for [Chromium](https://chromium-review.googlesource.com/c/chromium/src/+/3109968).
+
+
+## CSS transforms
+
+Support for `transform: perspective(none)` is now supported in 
+[Chromium](https://chromestatus.com/feature/5687325523705856), 
+[Gecko](https://bugzilla.mozilla.org/show_bug.cgi?id=1725207) and 
+[WebKit](https://trac.webkit.org/changeset/285255/webkit). This will make it easier to do 
+animations between a perspective and no perspective.
+
+In Chromium, `transform-style: preserve-3d` (which allows child elements to participate in the same 
+3D scene) and the `perspective` property (which applies a perspective transform to child elements) 
+are now [aligned with the spec](https://chromestatus.com/feature/5640541339385856) by making them 
+apply only to child elements.
+
+The big increase in the [scores](https://wpt.fyi/compat2021?feature=css-transforms) for CSS 
+transforms for all browsers is mainly due to improvements to the test suite, where some incorrect 
+tests have been fixed or removed. This makes it easier to understand the remaining interoperability 
+problems, and avoids regressions in the future.
+
+
+## Conclusion
+
+We are grateful for the work that everyone has put in to end the year with many improvements to the 
+score as well as better testing infrastructure. `aspect-ratio` was a long requested feature from 
+web developers and it is now supported in all browsers. Growth in use of flexbox and grid layout is 
```

Comment:
The sentence starting with "Growth" doesn't quite parse.

Reply:
Agreed. @kosamari I'll reword it as:

> And the growing use flexbox and grid layout is now better supported thanks to many improvements made during 2021.

Reply:
Hmm, it's not the *growth* that's better supported, it's the features themselves. Instead:

> Use of flexbox, grid and `position: sticky` are all growing, and these features are now better supported across browsers thanks to many improvements made during 2021.

Added `position: sticky` after looking at https://chromestatus.com/metrics/feature/timeline/popularity/3354